src/MacroTools/ObjectiveSystem/Objectives/UnitBased/ObjectiveKillXYUnit.cs

```diff
@@ -7,13 +7,13 @@ namespace MacroTools.ObjectiveSystem.Objectives.UnitBased
   /// <summary>
   ///   Completes when the quest holder has killed a certain number of units of a certain type.
   /// </summary>
-  public sealed class ObjectiveKillUnitType : Objective
+  public sealed class ObjectiveKillXYUnit : Objective
```

Comment:
This rename is quite odd - can you explain?

Reply:
Oh, this is me being stupid and forgot to reverse it. I didnt know that the killcount quest existed, il take care of that